storage/csi-cloning.md

```diff
@@ -0,0 +1,124 @@
+---
+title: CSI Cloning
+authors:
+  - "@chuffman"
+reviewers:
+  - "@gnufied”
+  - “@jsafrane”
+approvers:
+  - TBD
+  - "@..."
+creation-date: 2019-09-05
+last-updated: 2019-09-12
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# csi-cloning
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs] 
+
+## Summary
+In the past storage drivers have been provided either in-tree or via external provisioners. With the move to advocating CSI as the way to deliver storage drivers, Red Hat needs to ensure the CSI drivers have comparable functionality to the in-tree drivers.
+
+For the purpose of this enhancement, a Clone is defined as a duplicate of an existing Volume that can be consumed as any standard Volume would be. The only difference is that upon provisioning, rather than creating a "new" empty Volume, the back end device creates an exact duplicate of the specified Volume.
+
+## Motivation
+
+* It has to be assumed that at some point in-tree drivers will be removed from Kubernetes, we need to continue to make the drivers available and CSI is the way to do this.
+* As in-tree drivers move to CSI, we need to ensure that there is no loss in functionality by moving to a CSI driver.
```

Comment:
Again, there is no "loss of functionality", cloning is only for CSI drivers.

Our epic says that cloning "has been requested by both the OCS and CNV teams", so this is our motivation. I would not be afraid to directly copy the epic here.

Reply:
I've updated this section to more closely mirror the epic and remove the previous text.